gateway/gateway_config.go

```diff
@@ -13,6 +13,7 @@ func defaultConfig() config {
 		LargeThreshold:      50,
 		Intents:             IntentsDefault,
 		Compress:            true,
+		Compression:         CompressionZstdStream,
```

Comment:
should we really default to zstd stream?
maybe it should default to `CompressionZlibPayload` like rn

Reply:
Most libraries use zstd stream by default, but ultimately your call :)